sdk/data/azappconfig/client_test.go

```diff
@@ -32,7 +32,11 @@ func TestClient(t *testing.T) {
 	require.NoError(t, err)
 	require.NotEmpty(t, client)
 
-	addResp, err2 := client.AddSetting(context.TODO(), key, &value, &AddSettingOptions{Label: &label, ContentType: &contentType})
+	addResp, err2 := client.AddSetting(context.TODO(), key, &AddSettingOptions{
```

Comment:
I use context.Background() typically. Would context.TODO() be a better choice? I typically only use those in examples.

Reply:
While the end result is the same, typically, it's `context.Background()` for tests and `context.TODO()` for examples.  I will clean this up.

Reply:
Done.